api/s2n.h

```diff
@@ -241,12 +241,32 @@ S2N_API extern int s2n_init(void);
 S2N_API extern int s2n_cleanup(void);
 
 /**
- * Create a new s2n_config object. This object can (and should) be associated with many connection objects.
+ * Creates a new s2n_config object and initializes it with default system certificates. This object
+ * can (and should) be associated with many connection objects.
  *
- * @returns returns a new configuration object suitable for associating certs and keys. 
+ * The returned config should be freed with `s2n_config_free()` after it's no longer in use by any
+ * connection.
+ *
+ * @returns A new configuration object suitable for configuring connections and associating certs
+ * and keys.
  */
 S2N_API extern struct s2n_config *s2n_config_new(void);
 
+/**
+ * Creates a new s2n_config object with minimal default options. This function has better
+ * performance than `s2n_config_new()` due to skipping expensive operations.
```

Comment:
Can you add this sentence to fit with the next paragraph? Like:

This function has better performance than `s2n_config_new()` because it does not load default system certificates into...

I just think it's weird to not specify what the expensive operations are.